### PR TITLE
feat(pairing): support private browsing mode for pairing status

### LINKED ISSUE #552: Pairing: Signal that it's a pairing flow to Firefox Desktop in fxa_status
> * Provide a signal for Firefox Desktop that fxa_status should be allowed for the pairing context (currently it is only for `service === sync`)

Follow up to https://github.com/mozilla/fxa-content-server/pull/6876

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #552

cc @eoger

### CODE REVIEW

File: packages/fxa-content-server/app/scripts/models/auth_brokers/base.js [hunk 1]
```diff
@@ -163,12 +167,18 @@ const BaseAuthenticationBroker = Backbone.Model.extend({
...
    * @private
    */
-  _fetchFxaStatus () {
+  _fetchFxaStatus (statusOptions={}) {
     const channel = this._notificationChannel;
-    return channel.request(channel.COMMANDS.FXA_STATUS, this.relier.pick('service'))
+    const isPairing = statusOptions.isPairing;
+
+    return channel.request(channel.COMMANDS.FXA_STATUS, {
+      isPairing,
```
Comment (vladikoff) [inline packages/fxa-content-server/app/scripts/models/auth_brokers/base.js]: @shane-tomlinson could you confirm this is okay? Blocking https://phabricator.services.mozilla.com/D29082

Comment (shane-tomlinson) [reply packages/fxa-content-server/app/scripts/models/auth_brokers/base.js]: Seems reasonable to me, mind adding some tests for it and I can r+?

Comment (vladikoff) [reply packages/fxa-content-server/app/scripts/models/auth_brokers/base.js]: Thank you!

**Comment by vladikoff** [comment]
@shane-tomlinson updated!

**Pull request merged**